bindings/python/src/datetime.cpp

```diff
@@ -55,6 +55,29 @@ struct time_duration_to_python
     }
 };
 
+struct time_point_to_python
+{
+    static PyObject* convert(lt::time_point pt)
+    {
+        using boost::chrono::system_clock;
+        using boost::chrono::duration_cast;
+        time_t const tm = system_clock::to_time_t(system_clock::now()
+            + duration_cast<system_clock::duration>(pt - lt::clock_type::now()));
```

Comment:
the reason to use absolute timestamps here (or at least in general) is to avoid having to post ``state_update_alert`` continuously. If these counters are relative, and constantly updates, a client might expect to be notified of those changes.

I'm not sure what printing ``pt`` will print. It's not really useful without also printing ``lt::clock_type::now()`` I think.

btw, do you understand why this isn't causing test failures on travis?